### ISSUE TITLE
binding for rotation is wrong in cinema mode

### ISSUE THREAD

**Issue by cheater** [phileday/whirligig-old-issues#18]
There is a binding to "turn left" and "turn right", but in cinema mode instead it changes the aspect ratio of the "cinema screen". Please enable rotation in cinema mode.

**Comment by cheater** [comment]
Most likely related to #28 and #7.

**Comment by phileday** [comment]
So this is to do with the function differences between Cinema and fisheye/barrel/custom.

This is more of a control naming issue. In the setup tilt should also read change aspect ratio in Cinema or something on those lines. I'll look into making this more clear.

**Comment by cheater** [comment]
Yeah, but that's not the point. The more important point is that there needs to be a way to turn left and right, like in #28.
